packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -59,7 +59,8 @@ const linkStyles = (props: IPivotStyleProps): IStyle[] => {
           cursor: 'pointer'
         },
         ':active': {
-          backgroundColor: palette.neutralLight
+          backgroundColor: palette.neutralLight,
```

Comment:
is there a semantic slot we could use?

@phkuo @aneeshack4 fyi

Reply:
Maybe the slots for actionLinks/Buttons? The borderless, backgroundless buttons.